frontend/portals/ProductDescriptionAfter/__snapshots__/index.spec.jsx.snap

```diff
@@ -0,0 +1,20 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Could you please explain me why did you renamed the Portals directory to "portals"?

I see this is only renamed here, but it's not changed in the extension config. Which means it would fail during the deployment on any case-sensitive machine.

Reply:
I updated directory name to reflect other directory casing. Discovered an issue in travis with unit tests failing because directory could not be found. Tests were not failing on my machine because it is not case sensitive. Updated extension config directory name